src/ruby/puppetserver-lib/puppet/server/master.rb

```diff
@@ -102,6 +102,14 @@ def terminate
     Puppet::Server::Config.terminate_puppet_server
   end
 
+  def getTasks(env)
+    environment = @env_loader.get(env)
+    unless environment.nil?
+      tasks_per_env =
+        Puppet::InfoService.tasks_per_environment(environment.name)
+    end
```

Comment:
Yeah, I haven't updated this part yet, wanted to leave the thread open for discussion on the nil part